src/components/MovieAdd/js/MovieAdd.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import Constants from '../../../constants';
+import MovieAddForm from './MovieAddForm';
+import { movieActions } from '../../../store/actions';
+import { connect } from 'react-redux';
+import '../css/MovieAdd.css';
+
+const MovieAdd = ({ modalTitle, closeAction, onSubmit }) => {
+
+  const availableGenres = useMemo(() => {
+    const defaultGenresNames = Constants.DEFAULT.map((genre) => genre.value).sort();
+    return defaultGenresNames.map((genre) => ({ label: genre, value: genre })).sort();
```

Comment:
Isn't sort redundant here?

Reply:
Good point, thanks for your note. I'll fix it in the next PR.